src/app/shared/routing/group-route.ts

```diff
@@ -16,33 +17,47 @@ export interface GroupRouteError {
   id?: string;
 }
 
-export function groupRoute(id: GroupId, path: string[]): GroupRoute {
-  return { contentType: 'group', id: id, path };
+
+export type GroupLike =
+  | { id: Group['id'], type?: string }
+  | Pick<User, 'groupId' | 'login'>;
+
+function isUser(group: GroupLike): boolean {
+  return ('login' in group && !!group.login) || ('type' in group && group.type === 'User');
```

Comment:
Which means that if you have a group without login and type, you might classify a group as non-user while you have no guarantee.  This is quite misleading as a dev might use one of the pipe/function thinking that it is handled correctly while some data is missing to determine whether it is a user or not.
It might be that the `GroupLike` type is incomplete... it might require a third option with the "caller" forces to user or non-user because he knows it.

Reply:
Fixed